studioShared/schemas/fields/customerCaseProjectInfo.ts

```diff
@@ -36,9 +36,10 @@ export const customerCaseProjectInfo = defineField({
     defineField({
       // TODO: We should be able to select the consultants from a list
       name: "consultants",
-      description: "The consultants for the project",
+      description:
```

Comment:
Do we need to use @variant.no or could we only use "amn"?

Reply:
We are not sure enough that they are unique across countries (e.g. mom@variant.no and mom@variant.se could be two different people). So going for the safer option now. The user experience may be a little worse, but that could be fixed with a dropdown/search solution (where the internal value does not matter to the user)